.github/workflows/code-review.yml

```diff
@@ -0,0 +1,41 @@
+name: ARC - CodeReview
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+    paths:
+      - 'pagopa-arc-be/**'
```

Comment:
i'm not sure the paths is necessary. This would exclude the changes of `build.gradle` file

Reply:
actually I thought I'd remove the path here because I don't want to limit the github action to a specific change